freqtrade/commands/data_commands.py

```diff
@@ -50,7 +50,13 @@ def start_download_data(args: Dict[str, Any]) -> None:
     exchange = ExchangeResolver.load_exchange(config['exchange']['name'], config, validate=False)
     markets = [p for p, m in exchange.markets.items() if market_is_active(m)
                or config.get('include_inactive')]
-    expanded_pairs = expand_pairlist(config['pairs'], markets)
+    if config.get('freqai') is not None:
+        assert config['freqai'].get('corr_pairlist'), "No corr_pairlist found in config."
```

Comment:
Should be removed now that we validate `corr_pairlist` in `conf_validatation.py`